File: trail1/정수의 조건 여부 3/numbers-condition-3.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int a = int.Parse(Console.ReadLine());
        string res = a % 13 == 0 || a % 19 == 0 ? "True" : "False";
        Console.WriteLine(res);
    }
}

File: trail1/정수의 조건 여부 3/numbers-condition-3.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    int a;
    cin >> a;
    string res = a % 13 == 0 || a % 19 == 0 ? "True" : "False";
    cout << res << " ";
    return 0;
}